src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import TodoContainer from "./components/TodoContainer";
+
+ReactDOM.render(
+  <React.StrictMode>
+    <TodoContainer />{" "}
+  </React.StrictMode>,
+  document.getElementById("root")
+);
```

Comment:
Although this may work, when running it in the browser, the DevTools console throws an error.

![review](https://user-images.githubusercontent.com/60033459/177400772-50d4d56c-42f6-4a6f-ad60-316f1bee8e19.png).

Perhaps you could use the command cited there: "createRoot", like this:
```
const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <TodoContainer />
  </React.StrictMode>,
);
```

Reply:
Hello @jevazquezb 😁 
Thanks for the review, the root method is now updated and working. 👍🏼